net/libtorrent-rasterbar11/Portfile

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           conflicts_build 1.0
+PortGroup           cxx11 1.1
```

Comment:
this PG is obsolete, please use ```compiler.cxx_standard   2011``` instead (ideally somewhere close to ```configure.cxxflags-append``` in the Portfile)